java-runtime-builder-app/src/main/java/com/google/cloud/runtimes/builder/Application.java

```diff
@@ -56,6 +56,20 @@
         .longOpt("server-runtimes-map")
         .desc("Mappings between supported jdk versions, server types, and docker images")
         .build());
+
+    CLI_OPTIONS.addOption(Option.builder("m")
+        .required()
```

Comment:
:+1: to `<BUILDER>`. 

Regarding the other thing, I'd kind of like to avoid making the args optional. It introduces a new failure mode to the program and adds complexity - I'd prefer to leave this alone for now.